app/timeline/particles.js

```diff
@@ -49,7 +49,7 @@ const App = () => {
                   type: "circle",
                 },
                 onHover: {
-                  enable: true,
+                  enable: false,
```

Comment:
Keep this true, was looking good
```suggestion
                  enable: true,
```